Gopkg.toml

```diff
@@ -84,3 +84,7 @@ required = [
 [[override]]
   revision = "411b2483e5034420675ebcdd4a55fc76fe5e55cf"
   name = "k8s.io/kube-openapi"
+
+[[constraint]]
+  name = "github.com/gorilla/csrf"
```

Comment:
what license?

Reply:
It's 3-clause BSD license, fully compatible with Apache 2.0